### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by alfi4000** [community/community#81637] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I have a Question I have a Dell PowerEdge r730xd and was wondering why the Web interface was on the devices who access the web interface laggy is that normal or what is the problem?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by alfi4000** [comment]
i have like 150 Loras inside and still 15 checkpoints and I am planning to add more daily I add new ones this is might be the cause!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @alfi4000 ! 

We're happy you're here. The Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Dell. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
